test/cql-pytest/test_tablets.py

```diff
@@ -107,3 +107,22 @@ def test_describe_initial_tablets(cql, skip_without_tablets):
     with new_test_keyspace(cql, ksdef) as keyspace:
         desc = cql.execute(f"DESCRIBE KEYSPACE {keyspace}")
         assert "and tablets = {'initial': 1}" in desc.one().create_statement.lower()
+
+
+def verify_tablets_presence(cql, keyspace_name, table_name, expected:bool=True):
+    res = cql.execute(f"SELECT * FROM system.tablets WHERE keyspace_name='{keyspace_name}' AND table_name='{table_name}' ALLOW FILTERING")
+    if expected:
+        assert res, f"{keyspace_name}.{table_name} not found in system.tablets"
+        assert res.one().tablet_count > 0, f"table {keyspace_name}.{table_name}: zero tablets allocated"
+    else:
+        assert not res, f"{keyspace_name}.{table_name} was found in system.tablets after it was dropped"
+
+# Test that when a tablets-enabled table is dropped, all of its tablets are dropped with it.
```

Comment:
Thanks, I think the test is much clearer now, and thanks for adding a comment. Next time please add a reference to the issue that each test reproduces (since you did write these tests to reproduce a specific issue!), but I won't insist now - this review has dragged on long enough.

Reply:
Added references to the issue in v4

Reply:
I don't see it here?

Reply:
This particular test, for a table that has no views or indices, has no issue.
It is working without this series.
The next tests reference the issue fixed in this series.